gnsslib/src/main/java/com/android/gpstest/util/Config.java

```diff
@@ -76,7 +79,7 @@ public boolean isAutoShareEnabled() {
 
     public void setGpsOnly(boolean gpsOnly) {
         Config.gpsOnly = gpsOnly;
-        prefs.edit().putBoolean(PREFS_GPS_ONLY, gpsOnly).commit();
+        prefs.edit().putBoolean(PREFS_GPS_ONLY, gpsOnly).apply();
```

Comment:
Same with all of these preference changes.  I am nervous there was a reason to use commit instead of apply.